src/executables/utility.h

```diff
@@ -1387,6 +1389,8 @@ typedef struct _ha_config
 #define CSQL_QUERY_COLUMN_ENCLOSURE_L		"enclosure"
 #define CSQL_LOADDB_OUTPUT_S			'd'
 #define CSQL_LOADDB_OUTPUT_L			"loaddb-output"
+#define CSQL_PAGE_SERVER_S                      34556
```

Comment:
```suggestion
#define CSQL_PAGE_SERVER_S                      10219
```

Reply:
did you mean 12020, as the previous one `CSQL_QUERY_COLUMN_ENCLOSURE_S` has the value 12019